backend/database-sqlite.js

```diff
@@ -96,7 +65,6 @@ export function readData (key: string): Promise<Buffer | string | void> {
 
 export function writeData (key: string, value: Buffer | string): Promise<void> {
   return new Promise((resolve, reject) => {
-    checkKey(key)
```

Comment:
Similar comment here. While we are still supporting `fs`, and even after, we do want to make sure improperly named keys are not being passed in.